### ISSUE TITLE
More examples ?

### ISSUE THREAD

**Issue by SrivastavaKshitij** [apache/incubator-mxnet#8581]
Hi There

I am trying to use mxnet and gluon for text classification using cnn. I can't find extensive examples for mxnet. All the gluon tutorials , show models which have been initialized as Net = gluon.nn.Sequential(). I would like to see examples which have been created using block or hybrid class. 

Can somebody redirect me to where I can find such examples

**Comment by SrivastavaKshitij** [comment]
This is the problem with mxnet. Just one example per problem. There is no extensive coverage for any topic.

**Comment by szha** [comment]
@apache/mxnet-committers: This issue has been inactive for the past 90 days. It has no label and needs triage.

For general "how-to" questions, our [user forum](https://discuss.mxnet.io/) (and [Chinese version](https://discuss.gluon.ai/)) is a good place to get help.

**Comment by anirudh2290** [comment]
@szha Can you please remove the "Need Triage" and add "Call for Contribution" "Example"

**Comment by szha** [comment]
@anirudh2290 I don't think this issue is actionable in the way the proposed label suggests yet.

@SrivastavaKshitij do you have a wish list of examples that you'd like to share?

**Comment by piyushghai** [comment]
@SrivastavaKshitij Here's a list of example models we have on MXNet : https://github.com/apache/incubator-mxnet/tree/master/example 

You can also have a look at our Model Zoo : https://mxnet.apache.org/model_zoo/index.html
Hope this helps!
